src/components/sections/BannerText/banner-text.scss

```diff
@@ -0,0 +1,47 @@
+@import "../../../styles/vendors/include-media.scss";
+
+[data-store-banner] {
+  padding: var(--space-6) var(--space-5);
+  text-align: center;
+  background-color: var(--color-secondary-4);
+
+  @include media(">=notebook") {
+    padding: var(--space-9) 273px;
+  }
+
+  [data-store-banner-content] {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  h2, p {
```

Comment:
The paragraph having the same style as the heading is a bit of a red flag for me. 

[On Figma](https://www.figma.com/file/cAXtiQGObWxamFVVhviQVM/Handoff-Homepage-v0.2?node-id=927%3A74363) there's a variation where the paragraph has a smaller font (16px vs the heading's 20px), but that's not our use case here.
![Screen Shot 2021-12-28 at 11 51 19](https://user-images.githubusercontent.com/381395/147578330-2c2dbd3a-a54e-4634-8236-d81a8ae4435e.png)

Could this be an indication that we should pass the `BatterText`s `title` as `Receive our news and promotions in advance. Enjoy and get 10% off your first purchase.` and do not pass the `caption`? This way we don't have to style the paragraph the same way as the heading.

Something like:
```jsx
/* Skip the `caption` prop */
<BannerText
  title="Receive our news and promotions in advance. Enjoy and get 10% off your first purchase."
  actionPath="/"
  actionLabel="Call to action"
/>
```
instead of the current:
https://github.com/vtex-sites/base.store/blob/ca403b3bad315be6dcd363c161e1bb7b3d223043/src/pages/index.tsx#L63-L68

Reply:
In handoff documentation they have differences between title and caption, but in the Alpha version the haven't, so i thought i should do this and change in next version
![Screen Shot 2021-12-28 at 12 07 02](https://user-images.githubusercontent.com/51174217/147579761-04c523f0-ee73-4661-a7dc-49b63caae1e1.png)

![Screen Shot 2021-12-28 at 12 07 13](https://user-images.githubusercontent.com/51174217/147579770-c80a36d9-1260-4331-a42d-9673213f1ca9.png)